docs/docsite/rst/playbooks_prompts.rst

```diff
@@ -27,6 +27,9 @@ Here is a most basic example::
         - name: "favcolor"
           prompt: "what is your favorite color?"
 
+.. note::
+   Prompts for "vars_prompt" variables will be skipped for any variable that is already defined on the command line. See :ref:`_passing_variables_on_the_command_line` in the /Variables/ chapter.
```

Comment:
or when running from a batch system (cron, tower, etc) when you don't have a tty